Lib/test/test_email/test_policy.py

```diff
@@ -237,6 +237,12 @@ def test_adding_default_policies_preserves_default_factory(self):
                          email.policy.EmailPolicy.header_factory)
         self.assertEqual(newpolicy.__dict__, {'raise_on_defect': True})
 
+    def test_non_ascii_chars_do_not_cause_inf_loop(self):
+        policy = email.policy.default + email.policy.strict
+        msg = email.message.EmailMessage()
```

Comment:
I think this doesn't need to be an `EmailMessage`, you can trigger this with just:

```python
policy = email.policy.default
policy.fold('Subject', 'ą' * 100)
```

Reply:
Thanks for the suggestion. Indeed, it's engough to just define the policy.